srcpkgs/samba/template

```diff
@@ -125,6 +132,45 @@ smbclient_package() {
 	}
 }
 
+ctdb_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - Clustered TDB support"
+	pkg_install() {
+		_tools="ctdb ctdbd ctdbd_wrapper ctdb_diagnostics ctdb_local_daemons ctdb_run_tests ctdb_run_cluster_tests"
+
+		for _t in ${_tools}; do
+			vmove usr/bin/${_t}
+		done
+
+		_man1="ctdb ctdb_diagnostics ctdbd ctdbd_wrapper"
+
+		for _f in ${_man1}; do
+			vmove usr/share/man/man1/${_f}.1
+		done
+
+		_man5="ctdb.conf ctdb-script.options ctdb.sysconfig"
+
+		for _f in ${_man5}; do
+			vmove usr/share/man/man5/${_f}.5
+		done
+
+		_man7="ctdb ctdb-statistics ctdb-tunables"
+
+		for _f in ${_man7}; do
+			vmove usr/share/man/man7/${_f}.7
+		done
+
+		vmove usr/lib/libctdb-event-client-samba4.so
+
+		vmove usr/libexec/ctdb
+		vmove usr/share/ctdb
+		vmove etc/ctdb
```

Comment:
Is it appropriate for package updates to clobber everything in `/etc/ctdb`, or should some of the files in `/etc/ctdb` be marked as `conf_files`?

Reply:
The files under `/etc/ctdb`, other than `ctdb.conf` and `nodes`, are scripts used for administering ctdb.

Now that I think about it, those scripts should probably go under `mutable_files`.

Reply:
Even if they are scripts, if you expect the user to modify them and you don't want to clobber them with every update, use `conf_files` instead of `mutable_files`. Files listed in `mutable_files` are still replaced during upgrades and removed when a package is removed, but XBPS ignores changes to the files when evaluating package integrity. Files listed in `conf_files` are removed or replaced if and only if their contents have not been changed by the user since installation time.